engine/src/main/scala/cromwell/webservice/CromwellApiHandler.scala

```diff
@@ -86,29 +83,20 @@ class CromwellApiHandler(requestHandlerActor: ActorRef) extends Actor with Workf
   private def error(t: Throwable)(f: Throwable => RequestComplete[_]): Unit = context.parent ! f(t)
 
   override def receive = {
-    case ApiHandlerWorkflowStatus(id) => requestHandlerActor ! OldStyleWorkflowManagerActor.WorkflowStatus(id)
-    case WorkflowManagerStatusSuccess(id, state) => context.parent ! RequestComplete(StatusCodes.OK, WorkflowStatusResponse(id.toString, state.toString))
-    case WorkflowManagerStatusFailure(_, e) =>
-      error(e) {
-        case _: WorkflowNotFoundException => RequestComplete(StatusCodes.NotFound, e)
-        case _ => RequestComplete(StatusCodes.InternalServerError, e)
-      }
+    case ApiHandlerWorkflowStatus(id) => ???
+    case WorkflowManagerStatusSuccess(id, state) => ???
+    case WorkflowManagerStatusFailure(_, e) => ???
```

Comment:
Any reason all this stuff can't be deleted?  These messages should now run to the MetadataBuilderActor, or in the case of call-level outputs and logs have been eliminated entirely.